shared-libs/tombstone-utils/test/tombstone-utils.js

```diff
@@ -0,0 +1,206 @@
+var sinon = require('sinon').sandbox.create(),
+    chai = require('chai'),
+    lib = require('../src/tombstone-utils');
+require('chai').should();
+
+var DB,
+    initedLib;
+
+describe('Tombstone Utils Lib', function() {
+  'use strict';
+
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  beforeEach(function() {
+    DB = { get: sinon.stub(), put: sinon.stub() };
+    initedLib = lib(Promise, DB);
+  });
+
+  describe('extractDocId', function() {
+    it('extracts docID from tombstoneID, if it matches the format, otherwise null', function() {
+      initedLib.extractDocId('docid____rev____tombstone').should.equal('docid');
+      chai.expect(initedLib.extractDocId('000011111')).to.equal(null);
+    });
+  });
+
+  describe('extractRev', function() {
+    it('extracts rev from tombstoneID, if it matches the format, otherwise null', function() {
+      initedLib.extractRev('docid____rev____tombstone').should.equal('rev');
+      chai.expect(initedLib.extractRev('Friday')).to.equal(null);
+    });
+  });
+
+  describe('extractDoc', function() {
+    it('returns tombstone property contents', function() {
+      var doc = { _id: 'some_tombstone', tombstone: { _id: 'some', foo: 'bar' } };
+      initedLib.extractDoc(doc).should.deep.equal({ _id: 'some', foo: 'bar' });
+      chai.expect(initedLib.extractDoc({})).to.equal(undefined);
+      chai.expect(initedLib.extractDoc(33)).to.equal(undefined);
+    });
+  });
+
+  describe('isTombstoneId', function() {
+    it('returns true for strings that match, false otherwise', function() {
+      initedLib.isTombstoneId('aaaa').should.equal(false);
+      initedLib.isTombstoneId('doc____rev____tombstone').should.equal(true);
+      initedLib.isTombstoneId('doc-rev-tombstone').should.equal(false);
```

Comment:
This is mixing up `expect` and `should` style. Consider standardising on `expect`?